docs/develop-on-a-remote-host.md

```diff
@@ -161,19 +161,12 @@ This section describes how to instantiate an AWS EC2 as the remote host.
 
 - Update system packages
    ```console
-   sudo apt update && sudo apt upgrade -y
-   ```
-- [Install Docker Engine on Ubuntu](https://docs.docker.com/engine/install/ubuntu/#install-using-the-repository)
-- Add your user account to docker group.
-   ```console
-   sudo usermod -aG docker ${USER}
+   sudo yum update -y
    ```
-- To apply the new group membership, log out of the server and back in. Run the
-  command `groups` to check that your user is a member of the group `docker`.
-- Verify that Docker Engine is installed correctly by running the hello-world
-  image.
+- Docker should already be readily available on the instance. Verify this
+by running any Docker command, e.g.
    ```console
-   docker run --rm hello-world
+   docker --version
    ```
 - Clone your fork in the home directory
 
```

Comment:
```suggestion

- Follow the [**Storing GitHub credentials on the EC2 instance**](https://sagebionetworks.jira.com/wiki/spaces/APGD/pages/2590244872/Service+Catalog+Instance+Setup#Storing-GitHub-credentials-on-the-EC2-instance) instructions to config your GitHub account
```

@vpchung I was not able to push changes. I need to follow the ^ instruction to make it work - adding EC2 instance SSH key to my Github account. If you don't end up needing this step, please ignore this suggestion.

Reply:
Good idea!